package.json

```diff
@@ -13,6 +13,7 @@
   "devDependencies": {
     "@types/jest": "^25.1.3",
     "@types/node": "^13.7.7",
+    "array-flat-polyfill": "^1.0.1",
```

Comment:
Can we move this as dependencies unless you think it should be a devdependencies. :)

Reply:
Totally... my bad. I ran `npm i -D ..` out of habit, I'll fix the PR

Reply:
done